CourseGrab/ViewControllers/Search/SearchViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SearchViewController.swift
+//  CourseGrab
+//
+//  Created by Daniel Vebman on 2/3/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Tactile
+
+class SearchViewController: UIViewController {
+
+    private var popRecognizer: InteractivePopRecognizer?
+    private let textField = UITextField()
+
+    override func viewDidLoad() {
+        view.backgroundColor = .white
+
+        // Setup navigationBar
+        
+        textField.frame.size.width = view.frame.width - 80
+        textField.attributedPlaceholder = NSAttributedString(
+            string: "Search for a course",
+            attributes: [
+                .foregroundColor: UIColor(white: 1, alpha: 0.5),
+                .font: UIFont._20Medium
+            ]
+        )
+        textField.font = ._20Medium
+
+        let backButton = UIButton(type: .system)
+        backButton.setImage(.backIcon, for: .normal)
+        backButton.on(.touchUpInside, back)
+
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        navigationItem.leftBarButtonItems = [
+            UIBarButtonItem(customView: backButton),
+            UIBarButtonItem(customView: textField)
+        ]
```

Comment:
i could be wrong but if you're setting the `leftBarButtonItems` already, do you need to set the `backBarButtonItem` too?

Reply:
Yeah, they are different because we actually want both. The `backBarButtonItem` is inserted automatically by the navigation controller when you push a new view controller onto the stack. I set it explicitly so that it would have just the arrow and not say "< Back" 

The `leftBarButtonItems` are separate, on its right.